README.md

```diff
@@ -60,3 +60,15 @@ The `solid-client-authn` libraries are compatible with [NSS](https://github.com/
 
 - [Using Inrupt Solid JavaScript Client Libraries to authenticate](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/authenticate/)
 - [Inrupt documentation Homepage](https://docs.inrupt.com/)
+
+## How to run the test client login app?
+
+```javascript
+git clone https://github.com/inrupt/solid-client-authn-js
+cd solid-client-authn-js
+npm i
```

Comment:
Not sure if you'd want `npm i` here or `npm ci` - @NSeydoux ...?

Reply:
I think you'd prefer `npm ci`, even though I'll make sure dependabot updates the dependencies if the demo app as well. Also, generally, we tend to use full names for the commands (e.g. `npm install`) in tutorials :)